python_modules/dagster-graphql/dagster_graphql/schema/pipelines/pipeline.py

```diff
@@ -622,6 +629,16 @@ def resolve_rootConcurrencyKeys(self, graphene_info: ResolveInfo):
             root_concurrency_keys.extend([concurrency_key] * count)
         return root_concurrency_keys
 
+    def resolve_hasRunMetricsEnabled(self, graphene_info: ResolveInfo):
+        if self.dagster_run.status in UNSTARTED_STATUSES:
+            return False
```

Comment:
hm i wonder if somebody would ever want to see the resource usage of a run container while its in STARTING but not in STARTED yet (not sure if that is even possible with the current implementation)

Reply:
I don't think it's feasible either; even if the started message somehow got processed after the run metric thread start, there is a few seconds delay before the metrics are ingested and shown by datadog.

Also, I am using the same logic in dagit to avoid spurious queries to the datadog api. I want to avoid somehow somehow spamming reload to attempt to get timeseries that don't exist and deplete our api rate limit.